utils/ssh/sshcmd.go

```diff
@@ -100,11 +100,11 @@ func (s *SSH) CmdAsync(host net.IP, hostEnv map[string]string, cmds ...string) e
 				return fmt.Errorf("failed to start command %s on %s: %v", cmd, host, err)
 			}
 
-			ReadPipe(stdout, stderr, s.AlsoToStdout)
+			errMsg := ReadPipe(stdout, stderr, s.AlsoToStdout)
```

Comment:
how do we read message from a chan ? i think it is no need to return it, just print it to stdout

Reply:
thx for your review, I will fix my code